Integrations/integration-Netskope.yml

```diff
@@ -222,9 +235,9 @@ script:
       - Last90Days
       description: TimePeriod
     - name: starttime
-      description: 'start time: Month Day, Year HH:MM:SS (e.g. December 31, 1999 11:59:59)'
+      description: 'Query start time: timestamp or dd-mm-yyyyTHH:MM:SSZ (e.g. 31-12-1999T11:59:59Z)'

```

Comment:
Are we not breaking BC ?

Reply:
the way it worked in the original format is by passing timestamp.
the original code passes the CLI argument directly to the request.
the current API accept only timestamp so the old format didn't work at all.
so if it didn't work anyway, why not use demisto standard for time string format...